datapackage/resource.py

```diff
@@ -5,9 +5,12 @@
 
 
 import os
+import json
 import six
 import six.moves.urllib as urllib
+from future.utils import raise_with_traceback
```

Comment:
I've never used `raise_with_traceback`, but it sounds interesting. Over `datapackage-py`, we've been using `six.raise_from` as a Python-version independent way of wrapping exceptions (like Py3's `raise XXX from exp`), but I wonder if `raise_with_traceback` is better (of it it's the same, in which case we should use only one).